meet-data/spa/1701/entries.csv

```diff
@@ -0,0 +1,91 @@
+Name,WeightClassKg,Division,Sex,Squat1Kg,Squat2Kg,Squat3Kg,BestSquatKg,Bench1Kg,Bench2Kg,Bench3Kg,BestBenchKg,Deadlift1Kg,Deadlift2Kg,Deadlift3Kg,BestDeadliftKg,TotalKg,BodyweightKg,Event,Equipment,Place
+Joel Tan Zheng How,60,T16-17,M,120,130,135,135,60,-65,-65,60,160,-175.5,-175.5,160,355,59.9,SBD,Wraps,1
+Gary Teo Shuan Yang,60,O,M,135,147.5,-160,147.5,80,90,97.5,97.5,160,175,-190,175,420,59.6,SBD,Wraps,1
+Kharil Amir Bin Rasid,60,J(20-23),M,135,-145,150,150,90,-100,100,100,180,190,-192.5,190,440,59.4,SBD,Wraps,1
+Amitabh Singh,67.5,M(50-54),M,-60,60,90,90,30,50,-65,50,60,90,120,120,260,65.9,SBD,Wraps,1
+Lahang Thamsuhang,67.5,T16-17,M,-100,105,-120,105,65,70,75,75,120,130,140,140,320,66.2,SBD,Wraps,1
+Nguyen Ngoc Tai,67.5,J(20-23),M,-110,120,-135,120,85,-90,-92.5,85,145,-152.5,157.5,157.5,362.5,64,SBD,Wraps,3
```

Comment:
This is probably last name first, right? I don't have any intuition for the order of these names...